content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: 'Gateway API First Beta'
+date: 2022-07-12
+slug: gateway-api-first-beta
+---
+
+**Author:** Shane Utt (Kong)
+
+We're pleased to announce that key Gateway API resources are graduating to
+beta with the `v0.5.0` release of the API. In addition to `GatewayClass`,
+`Gateway` and `HTTPRoute` graduating to beta, this release introduces several
+new concepts to the API that we'll cover below.
+
+## What is Gateway API
+
+Gateway API is a collection of resources centered around `Gateway` resources
+(which represent the underlying network gateways / proxy servers) to enable
+robust Kubernetes service networking through expressive, extensible and
+role-oriented interfaces that are implemented by many vendors and have broad
+industry support.
+
+Originally conceived as a successor to the well known `Ingress` API, the
+benefits of Gateway API include (but are not limited to) explicit support for
+many commonly used networking protocols (e.g. `HTTP`, `TLS`, `TCP`, `UDP`) as
+well as tightly integrated support for Transport Layer Security (TLS). The
+`Gateway` resource in particular enables implementations to manage the lifecycle
+of network gateways as a native Kubernetes API.
```

Comment:
Gateway is _not_ a native Kubernetes API. It's an official extension API (or it might be semi-official - see following questions).

- Does Gateway and friends go through formal API review? If it does, that review is a benefit to cluster operators who want to rely on it.
- Similarly, does Gateway now benefit from following the official [Kubernetes deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/)? Whatever the answer is, we should make that clear in the graduation announcement.

Reply:
:thought_balloon: if Gateway and friends are an official, API-reviewed Kubernetes API, then for beta onwards we can also document that API within https://kubernetes.io/docs/

Reply:
For now just so that we don't get caught up on this I'll just remove this language:
```suggestion
of network gateways as a Kubernetes API.
```